pinot-broker/src/main/java/org/apache/pinot/broker/broker/BrokerServerBuilder.java

```diff
@@ -64,33 +62,21 @@ public BrokerServerBuilder(Configuration config, RoutingTable routingTable, Time
         config.getLong(Broker.CONFIG_OF_DELAY_SHUTDOWN_TIME_MS, Broker.DEFAULT_DELAY_SHUTDOWN_TIME_MS);
     _routingTable = routingTable;
     _timeBoundaryService = timeBoundaryService;
-    _queryQuotaManager = queryQuotaManager;
     _accessControlFactory = AccessControlFactory.loadFactory(_config.subset(Broker.ACCESS_CONTROL_CONFIG_PREFIX));
     _metricsRegistry = new MetricsRegistry();
     MetricsHelper.initializeMetrics(config.subset(Broker.METRICS_CONFIG_PREFIX));
     MetricsHelper.registerMetricsRegistry(_metricsRegistry);
     _brokerMetrics =
-        new BrokerMetrics(config.getString(Broker.CONFIG_OF_METRICS_NAME_PREFIX, Broker.DEFAULT_METRICS_NAME_PREFIX), _metricsRegistry,
+        new BrokerMetrics(config.getString(Broker.CONFIG_OF_METRICS_NAME_PREFIX, Broker.DEFAULT_METRICS_NAME_PREFIX),
+            _metricsRegistry,
             !_config.getBoolean(Broker.CONFIG_OF_ENABLE_TABLE_LEVEL_METRICS, !Broker.DEFAULT_METRICS_GLOBAL_ENABLED));
     _brokerMetrics.initializeGlobalMeters();
-    _brokerRequestHandler = buildRequestHandler();
+    _brokerRequestHandler =
```

Comment:
I suggest you keep the old config, but report a warning that it has been removed. 
Do we have release notes in 0.2.0 that this config is deprecated? If not, let us modify the release notes